packages/react/src/components/dropdown/select/Select.stories.tsx

```diff
@@ -140,6 +140,13 @@ DisabledOptions.storyName = 'With disabled options';
 export const Icon = (args) => <Select {...args} icon={<IconFaceSmile />} />;
 Icon.storyName = 'With icon';
 
+export const MultiselectWithIcon = (args) => <Select {...args} icon={<IconFaceSmile />} />;
```

Comment:
Should we use some icon that tells more about the options as the documentation states? We could use a location-related icon here, etc.

Reply:
- [ ] Use location icon